src/reducer.js

```diff
@@ -27,6 +27,10 @@ function reducer(
     fetchedLastClaimAt: false,
     errorLastClaimAt: null,
     lastClaimAt: {},
+    fetchingSameDiagnosisClaim: false,
+    fetchedSameDiagnosisClaim: false,
+    sameDiagnosisClaim: {},
```

Comment:
You're using here '{}', but in reducers you're using 'null'. Please, use consistent approach (decide whether you want to stay with '{}' or 'null').